spacy/tests/lang/grc/test_text.py

```diff
@@ -0,0 +1,23 @@
+import pytest
```

Comment:
Here, be sure to also add `__init__.py` in this directory or `pytest` will run into problems.

Reply:
To clarify, `__init__.py` should be an empty file.

Reply:
Does the __init__.py go empty? The one for Spanish has no text in it...

Reply:
figured out. thanks.